app/Providers/RouteServiceProvider.php

```diff
@@ -29,6 +29,12 @@ class RouteServiceProvider extends ServiceProvider
      */
     public const HOME = 'horizon';
 
+    public function boot()
+    {
+        parent::boot();
+        Route::model('departureReleaseRequest', DepartureReleaseRequest::class);
```

Comment:
I'm assuming the model name doesn't match well in the routes file for this?

Reply:
Yeah I think it didnt like this when I was first setting up, fixed now.